cmake/onnxruntime_python.cmake

```diff
@@ -281,6 +281,14 @@ file(GLOB onnxruntime_python_datasets_data CONFIGURE_DEPENDS
     "${ONNXRUNTIME_ROOT}/python/datasets/*.onnx"
 )
 
+# Files needed to convert ONNX model to ORT format
+set(onnxruntime_ort_format_model_conversion_srcs
+    ${REPO_ROOT}/tools/python/util/convert_onnx_models_to_ort.py
+    ${REPO_ROOT}/tools/python/util/logger.py
```

Comment:
maybe we should have some CI test that calls the script from the Python package to verify that we don't miss any dependencies that might be added in the future. that can be in a separate change.

Reply:
Will add in separate PR.